src/tests/test_testing.py

```diff
@@ -35,7 +35,7 @@ def func(f):
         "chrome": "nlopt set_min_objective triggers a fatal runtime error in chrome 89 see #1493",
     },
 )
-def some_func():
+def some_func(f):
```

Comment:
f was undefined, hopefully not intentional?

Reply:
Nope, definitely doesn't look intentional. Of course we never plan to call this function so it doesn't matter too much.